core/chains/evm/client/errors_test.go

```diff
@@ -379,6 +380,8 @@ func Test_Eth_Errors_Fatal(t *testing.T) {
 		{"Failed to serialize transaction: max priority fee per gas higher than 2^64-1", true, "zkSync"},
 		{"Failed to serialize transaction: oversized data. max: 1000000; actual: 1000000", true, "zkSync"},
 
+		{"failed to forward tx to sequencer, please try again. Error message: 'invalid sender\"", true, "Mantle"},
```

Comment:
`'invalid sender\"` has different types of quotes. `'` vs `"` . Is it relevant?

Reply:
Good catch, just updated for consistency. Either way both will be captured so it doesn't change the expected result.